chia/wallet/wallet_coin_store.py

```diff
@@ -12,11 +13,16 @@
 from chia.util.lru_cache import LRUCache
 from chia.util.misc import UInt32Range, UInt64Range, VersionedBlob
 from chia.util.streamable import Streamable, streamable
-from chia.wallet.util.query_filter import AmountFilter, CoinRecordOrder, FilterMode, HashFilter
+from chia.wallet.util.query_filter import AmountFilter, FilterMode, HashFilter
 from chia.wallet.util.wallet_types import CoinType, WalletType
 from chia.wallet.wallet_coin_record import WalletCoinRecord
 
 
+class CoinRecordOrder(IntEnum):
+    confirmed_height = 1
+    spent_height = 2
```

Comment:
I think we make constants like these all upper case

Reply:
We have it mixed. Upper case, lower case, snake case, camel case.. And, it's unrelate to this PR and wasn't introduced here. @ytx1991 i saw you changed it already in the last commit but i think you should revert the change again also because now tests will fail anyway because it reflects the name of the coin store table and the name of the enum entries are used in the query.

Reply:
Reverted